packages/storage/src/implementation/requests.ts

```diff
@@ -279,10 +277,13 @@ export function metadataForUpload_(
   return metadataClone;
 }
 
+/**
+ * Prepare RequestInfo for resumable uploads as Content-Type: multipart.
```

Comment:
```suggestion
 * Prepare RequestInfo for uploads as Content-Type: multipart.
```
It is used for both resumable and non-resumable uploads.

Reply:
Thanks, fixed.